beacon-chain/forkchoice/protoarray/types.go

```diff
@@ -40,6 +42,13 @@ type Node struct {
 	graffiti       [32]byte    // graffiti of the block node.
 }
 
+// optimisticStore defines a structure that tracks the tips of the fully
+// validated blocks tree.
+type optimisticStore struct {
+	tips map[[32]byte]types.Slot
```

Comment:
Given the optimistic in `optimisticStore`. Hard to know at the first glance whether `tips` are validated or pending. Maybe `validatedTips` or something else